Item.cpp

```diff
@@ -0,0 +1,49 @@
+// Item.cpp
+// Sheng Fang, Rabiya Majeed, & Zulaikha Zakiullah
+// This instantiates the item class. Item stats should be read off a txt. file (?)
+
+#include "Item.h"
+
+Item::Item(std::string st) {
+
+	// Splits string by words and assigns the stats from array
+	std::stringstream ssline(st);
+	std::string* stats = new std::string[5]{};
+
+	int i = 0;
+
+	while (ssline.good() && i < 5) {
+		ssline >> stats[i];
+		++i;
+	}
+
+	type = stats[0];								// Specifies what class of item it is
+	itemID = std::stoi(stats[1]);					// Item name/more specific 
+	durability = std::stoi(stats[2]);
+	rarity = std::stoi(stats[3]);
+	value = std::stoi(stats[4]);					// Value of the item determines its price

```

Comment:
value is a double so I think you should use std::stod

Reply:
Oh yeah thanks, missed that